### DISCUSSION TITLE
macOS Monterey uses port 5000 for Airplay reception

### DISCUSSION THREAD

**Discussion by TradingPlacesResearch** [benbusby/whoogle-search#592] [Q&A] [answered]
Like the title says. Took me way too long to figure this out after updating because port scan said Control Center was using it, but it’s AirPlay reception. Something else in Control Center is listening on port 7000 too

I tried using port 5001 for Whoogle but that wouldn’t work. Workaround is to turn off AirPlay reception which I don't use on this computer anyway. System Preferences -> Sharing 

Just curious, is there any way to get Whoogle running on a different port than 5000?

**Comment by Albonycal** [comment, marked as answer]
I assume you are running it using the `run.sh` 
you can change the line 29 in the run.sh 
` --port "${PORT:-"${EXPOSE_PORT:-5000}"}"`
to the port of your liking

**Reply by TradingPlacesResearch** [reply]
I will try that, I was changing it in Docker. Thanks!
